deployments/fpga_plugin/build-initcontainer-image.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -xe
+
+# build CRI hook
+HOOK=fpga_crihook
+make -C ../../ $HOOK
+cp ../../cmd/$HOOK/$HOOK ./intel-fpga-crihook
+
+# build initcontainer image
+WORKSPACE=`realpath .`
+IMG="intel-fpga-initcontainer"
+TAG=$(git rev-parse HEAD)
+
+BUILD_ARGS='--rm'
+for var in ftp_proxy http_proxy https_proxy; do
+    BUILD_ARGS="${BUILD_ARGS} --build-arg $var=`echo ${!var}`"
+done
+
```

Comment:
before build, it would be good to check presense of archive `a10_gx_pac_ias_1_1_pv_rte_installer.tar.gz`, and if not found, some error message that will show URL where it should be downloaded from.